core/src/main/java/org/elasticsearch/common/io/stream/StreamInput.java

```diff
@@ -762,4 +785,5 @@ public static StreamInput wrap(byte[] bytes) {
     public static StreamInput wrap(byte[] bytes, int offset, int length) {
         return new InputStreamStreamInput(new ByteArrayInputStream(bytes, offset, length));
     }
+
```

Comment:
It's not just you. Some classes even have a blank line between the opening and the body, but not the body of the class and the closing.